src/ui/groups/detail/GroupDetailTabs.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { TabContainer, Nav, NavItem, TabContent, TabPane } from 'patternfly-react';
+
+import GroupDetailTabPagesContainer from 'ui/groups/detail/GroupDetailTabPagesContainer';
+import GroupDetailTabUsersContainer from 'ui/groups/detail/GroupDetailTabUsersContainer';
+import GroupDetailTabWidgetTypesContainer from 'ui/groups/detail/GroupDetailTabWidgetsContainer';
+import GroupDetailTabContentsContainer from 'ui/groups/detail/GroupDetailTabContentsContainer';
+
+const GroupDetailTabs = () => (
+  <div>
+    <TabContainer className="GroupDetailTabs__container" id="group-detail-tabs" defaultActiveKey={1}>
```

Comment:
Did you check out components `Tabs` and `Tab` from patternfly-react ? You should be able to achieve the same result with something like

```
<Tabs>
  <Tab eventKey={1} title={formattedText('app.pages')}>
    <GroupDetailTabPagesContainer />
  </Tab>
{/* ... etc ... */}
</Tabs>
```

Reply:
i follow the patternfly react, because if use bootstrap the ui is will different